src/read_csv.py

```diff
@@ -0,0 +1,20 @@
+import csv
+
+
+def read_swaps_from_csv():
+    with open('data/swaps_data_from_router.csv', newline='') as f:
+        reader = csv.reader(f)
+        data = list(reader)
+        orders = dict()
+        for order in data:
+            path_data = order[5].split("'")
+            path_data = path_data[:-1]
+            if(len(path_data) > 1):
+                block_number = int(order[0])
+                list_start = orders[block_number] if block_number in \
+                    orders else list()
+                list_start.append(
+                    {"sellToken": path_data[1],
```

Comment:
Not really sure about this parsing: isn't the sell token the first element (i.e. index=0) in the "path" list?

Reply:
If it helps, I have coded this yesterday to parse this CSV:

Reply:
I also tried: ast.literal_eval(path) yesterday, but it did not work. I guess, because it failed already in the first line, which you are excluding.
Because I was not using ast.literal_eval(path), I did it in an uncle way requiring me to take the second element, as the first one was a `[`.
Anyways. Thanks a lot for the code snippet. I used it.